lib/isotovideo_interface.pm

```diff
@@ -0,0 +1,17 @@
+package isotovideo_interface;
+use strict;
+use utf8;
+use warnings;
+
+sub VERSION {
+    my ($package, $version) = @_;
+
+    qx/isotovideo --version/ =~ /interface v(\d+)/;
```

Comment:
I almost had a heart attack xD, didn't remember the interface version :P change done in (https://github.com/os-autoinst/os-autoinst/pull/904)